Sources/Orbit/Components/Dialog.swift

```diff
@@ -22,14 +22,15 @@ public struct Dialog<Content: View>: View {
                     .accessibility(.dialogTitle)
 
                 Text(description)
-                    .foregroundColor(.inkNormal)
+                    .textColor(.inkNormal)
                     .accessibility(.dialogDescription)
             }
 
             content
 
             VStack(spacing: .xSmall) {
                 buttons
+                    .textColor(nil)
```

Comment:
I see it is mentioned in the updated doc comments - so is the new situation better than the old one in this regard?

Reply:
It is now much more flexible, as it allows us to distinguish overrides in the middle of the component hierarchy.
Now the behaviour is consistent with the native `foregroundColor` modifier - the `nil` signals that any outer environment overrides should be ignored and the "original embedded" colors of the component should be used.

In this example, we are saying: "even though buttons colors can be overridden, we do not want to do that in this component, because we want to allow any `textColor` override only for dialog `title` and `description`"

From the [docs](https://developer.apple.com/documentation/swiftui/view/foregroundcolor(_:)):

> Pass nil to remove any custom foreground color and to allow the system or the container to provide its own foreground color.